pandas/core/arrays/base.py

```diff
@@ -583,15 +588,23 @@ def argmin(self):
         --------
         ExtensionArray.argmax
         """
-        return nargminmax(self, "argmin")
+        if not skipna and self.isna().any():
+            return -1
```

Comment:
As I mentioned in my previous comment, I don't think we want this behaviour of returning -1 on the ExtensionArrays. We should first discuss what behaviour we want (https://github.com/pandas-dev/pandas/issues/33942), but IMO it's definitely not -1. 

So short term, I propose to keep this logic on the Series as you had it before

Reply:
I see. Sorry I didn't understand your comment earlier. I agree.